packages/sampler/stories/qa/Scroller.js

```diff
@@ -69,6 +69,36 @@ class ScrollerWithResizable extends React.Component {
 	}
 }
 
+class ScrollerLastFocusedItem extends React.Component {
+	componentDidMount () {
+		this.blurEvent = document.createEvent('Event');
+		this.blurEvent.initEvent('blur', true, true);
+		this.focusEvent = document.createEvent('Event');
+		this.focusEvent.initEvent('focus', true, true);
+	}
+
+	handleScrollStop = () => {
+		window.dispatchEvent(this.blurEvent);
+		window.dispatchEvent(this.focusEvent);
+	}
+
+	render () {
+		return (
+			<Scroller
+				focusableScrollbar
+				verticalScrollbar="visible"
+				style={{height: ri.scale(200)}}
+				onScrollStop={this.handleScrollStop}
+			>
+				<Item>Item</Item>
+				<Item>Item</Item>
+				<Item>Focus me and press right</Item>
```

Comment:
Suggest moving this item to 2nd place and change text to 'focus me, press right, then select on down arrow'
Change 3rd item to say 'Focus should return here'.  This will clarify that the focused item has scrolled off the viewport and that the new focused item should be the topmost.